Src/BootCamp.Chapter/UserExistsException.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    /// <summary>
+    /// Throws an exception when a new user with the same username as an existing user tries to be registered
+    /// </summary>
+    class UserExistsException : Exception
```

Comment:
Codacy found an issue: [Make this exception 'public'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5768165)